schema.sql

```diff
@@ -1,36 +1,39 @@
 CREATE TABLE Ingredient (
-  id integer PRIMARY KEY AUTOINCREMENT,
-  name varchar(255) NOT NULL,
-  measure_unit varchar(255) NOT NULL,
-  available DECIMAL(10,5) NOT NULL
+  id INTEGER PRIMARY KEY AUTO_INCREMENT,
```

Comment:
should be `AUTOINCREMENT` instead of `AUTO_INCREMENT` (see [here](https://sqlite.org/autoinc.html))